gf-webapp/src/component/pageBorder/header/burger/burgerContent/BurgerContent.tsx

```diff
@@ -0,0 +1,30 @@
+import {useDictionary} from "src/logic/DictionaryContext/useDictionary";
+import {BurgerLink} from "src/component/pageBorder/header/burger/burgerContent/burgerLink/BurgerLink";
+import styles from "src/component/pageBorder/header/burger/burgerContent/BurgerContent.module.scss";
+
+export const BurgerContent = () => {
+  const navigation = useDictionary().dictionary.navigation;
+
+  return (
+    <div className={styles.burger_content}>
+      <ul>
+        <BurgerLink
+          to="/"
+          value={navigation.rooms}
+        />
+        <BurgerLink
+          to="/rooms"
+          value={navigation.main}
+        />
```

Comment:
Swap the values. "/" leads to the main, not rooms

Reply:
Done